src/common/options/mds.yaml.in

```diff
@@ -1166,6 +1166,24 @@ options:
   services:
   - mds
   with_legacy: true
+# Max number of slow ops to track
+- name: mds_op_history_slow_op_size
+  type: uint
+  level: advanced
+  desc: maximum size for list of historical slow operations
+  default: 20
+  services:
+  - mds
+  with_legacy: true
+# Track the op if over this threshold
+- name: mds_op_history_slow_op_threshold
+  type: uint
+  level: advanced
+  desc: track the op if over this threshold
+  default: 10
+  services:
+  - mds
+  with_legacy: true
```

Comment:
there are `osd_op_history_slow_op_size` and `osd_op_history_slow_op_threshold` in `global.yaml.in` while `mon.yaml.in` also have these values, these values are set in `OSD::OSD()` and `Monitor::Monitor()` respectively, these should be sufficient to set them. Although you mentioned in commit message "mds op_tracker have no set slow op threshold, and history_slow_op_threshold default value is 0, cause to mds have slow op alway." Is it some sort of bottleneck with MDS you're facing due to this?

Reply:
We have slow op counter in TrackedOp, you can see: https://github.com/ceph/ceph/pull/52068. If we no set this value, this metrics will continuously growing.

Reply:
Hmm, Interesting. So, `op_history_slow_op_threshold` and `op_history_slow_op_size` are part of OSD and Monitor, and with this PR, MDS too. Let's say all of them set different values for `op_history_slow_op_threshold` and `op_history_slow_op_size`, in this case do we have any overriding mechanism? Else it is possible that a daemon's values might affect another, right?

Reply:
mds, osd, mon and every ceph daemon have self op_tracker object, each op_tracker manage their own configurations without affecting each other

Reply:
Ah, got it.

Reply:
I had a misconception, nvm